lte/gateway/c/core/oai/test/s1ap_task/test_s1ap_state_converter.cpp

```diff
@@ -50,29 +50,38 @@ TEST_F(S1APStateConverterTest, S1apStateConversionSuccess) {
   s1ap_state_t* init_state = create_s1ap_state();
   s1ap_state_t* final_state = create_s1ap_state();
 
-  enb_description_t* enb_association = s1ap_new_enb();
-  enb_association->sctp_assoc_id = assoc_id;
-  enb_association->enb_id = 0xFFFFFFFF;
-  enb_association->s1_state = S1AP_READY;
+  oai::EnbDescription* enb_association = s1ap_new_enb();
+  enb_association->set_sctp_assoc_id(assoc_id);
+  enb_association->set_enb_id(0xFFFFFFFF);
+  enb_association->set_s1_enb_state(magma::lte::oai::S1AP_READY);
 
   // filling ue_id_coll
-  enb_association->ue_id_coll.insert(1, 17);
-  enb_association->ue_id_coll.insert(2, 25);
+  magma::proto_map_uint32_uint64_t ue_id_coll;
+  ue_id_coll.map = enb_association->mutable_ue_id_map();
+  ue_id_coll.insert(1, 17);
+  ue_id_coll.insert(2, 25);
 
   // filling supported_tai_items
-  supported_ta_list_t* enb_ta_list = &enb_association->supported_ta_list;
-  enb_ta_list->list_count = 1;
-  enb_ta_list->supported_tai_items[0].tac = 1;
-  enb_ta_list->supported_tai_items[0].bplmnlist_count = 1;
-  enb_ta_list->supported_tai_items[0].bplmns[0].mcc_digit1 = 1;
-  enb_ta_list->supported_tai_items[0].bplmns[0].mcc_digit2 = 1;
-  enb_ta_list->supported_tai_items[0].bplmns[0].mcc_digit3 = 1;
-  enb_ta_list->supported_tai_items[0].bplmns[0].mnc_digit1 = 1;
-  enb_ta_list->supported_tai_items[0].bplmns[0].mnc_digit2 = 1;
-  enb_ta_list->supported_tai_items[0].bplmns[0].mnc_digit3 = 1;
 
+  magma::lte::oai::SupportedTaList* enb_ta_list =
+      enb_association->mutable_supported_ta_list();
+  enb_ta_list->set_list_count(1);
+
+  for (int tai_idx = 0; tai_idx < enb_ta_list->list_count(); tai_idx++) {
+    magma::lte::oai::SupportedTaiItems* supported_tai_items =
+        enb_ta_list->add_supported_tai_items();
+    supported_tai_items->set_tac(1);
+
+    supported_tai_items->set_bplmnlist_count(1);
+#define PLMN_BYTES 7
```

Comment:
Why are we redefining this? Is it not available in the existing headers?

Reply:
Thanks for bringing into notice. Removed the definition from here